bin/gollum

```diff
@@ -204,7 +204,14 @@ MSG
   opts.separator ''
   opts.separator '  Development:'
 
-  opts.on('--development-assets', 'For development purposes only. Use local stylesheets and javascript instead of the packaged assets.') do
+  opts.on('--development-assets [PATH]', 'For development purposes only. Use assets in PATH (default: `node_modules` in the gollum project root) instead of the packaged assets.') do |path|
+    if path
+      module Precious
+        module Assets
+        end
+      end
+      ::Precious::Assets.const_set('NODE_MODULES', path)
+    end
```

Comment:
I wonder if there's a way we can provide this flag to developers without advertising a flag to end users.

I would personally be happy to use an "undocumented" environment variable instead. i.e.:

```
GOLLUM_DEV_ASSETS=path/to/assets bundle exec gollum ~/my-wiki
```

Reply:
I also like this idea. We could manually filter out the option [here](https://github.com/gollum/gollum/blob/master/bin/gollum#L174)?

Reply:
Great idea!

Reply:
This would have become a bit hackish as there is no clean way of turning  the`opts` object into an array of option-like objects -- we can only get an array of the lines of the help text. This means we'd have to filter out each line in a description separately. So I opted to leave it in the help text, as it's clearly marked as a development option. Since it is marked as such, we needn't give full documentation for the option. So I've just implemented your idea of the `GOLLUM_DEV_ASSETS` environment variable without documenting it.